system-addon/lib/ActivityStream.jsm

```diff
@@ -97,6 +97,10 @@ const PREFS_CONFIG = new Map([
     title: "Show the Search bar",
     value: true
   }],
+  ["showSnippets", {
```

Comment:
@k88hudson We'll need to update the pref string [here](https://github.com/mozilla/activity-stream/blob/master/system-addon/lib/TelemetryFeed.jsm#L33) in the TelemetryFeed.jsm, so that it can set the correct value for `user_prefs` in the telemetry.